Doc/library/re.rst

```diff
@@ -1565,21 +1565,27 @@ search() vs. match()
 
 .. sectionauthor:: Fred L. Drake, Jr. <fdrake@acm.org>
 
-Python offers two different primitive operations based on regular expressions:
-:func:`re.match` checks for a match only at the beginning of the string, while
-:func:`re.search` checks for a match anywhere in the string (this is what Perl
-does by default).
+Python offers different primitive operations based on regular expressions:
+
++ :func:`re.search` checks for a match anywhere in the string,
+  (this is what Perl does by default)
++ :func:`re.fullmatch` checks for entire string to be a match
++ :func:`re.match` checks for a match only at the beginning of the string
+
 
 For example::
 
-   >>> re.match("c", "abcdef")    # No match
-   >>> re.search("c", "abcdef")   # Match
-   <re.Match object; span=(2, 3), match='c'>
+   >>> re.match("c", "cdef")    # match
+   <re.Match object; span=(0, 1), match='c'>
+   >>> re.search("c", "cdef")   # Match
+   <re.Match object; span=(0, 1), match='c'>
+   >>> re.fullmatch("c", "cdef")  # No Match
 
 Regular expressions beginning with ``'^'`` can be used with :func:`search` to
 restrict the match at the beginning of the string::
 
    >>> re.match("c", "abcdef")    # No match
+   >>> re.fullmatch("c", "abcdef")  # No Match
```

Comment:
I would not add this. This section is trying to show the differences between `search` and `match`. Adding a `fullmatch` example just confuses things.

Reply:
yes sir

Reply:
Please delete the `fullmatch` line here.